### ISSUE TITLE
New master branch

### ISSUE THREAD

**Issue by stigrj** [MRChemSoft/mrchem#101]
I think it's time to promote the `new-mrcpp` branch to become the new `master`. The current `master` has gone stale and we have now caught up with all features in the new one (MPI support is still missing, though).

The only divergent development that has happened on `master` is Susi's fixes for boost, blas and eigen, but I'm not sure how this should be incorporated into the new super-build approach: boost is only used by getkw and blas should only be used by eigen, but I guess both must be picked up at mrchem level and passed on?

**Comment by robertodr** [comment]
I agree with the proposed change, but I'd ask you to wait a little longer until my changes in #96 are finalized. Regarding your questions:
- Eigen should be picked up by MRChem and passed along to submodules as needed.
- Boost should be picked up by Getkw. The only dependency is the `boost/any.hpp` header file, so once Getkw it's compiled, MRChem doesn't need to care about Boost at all. I need to encode this properly into a CMake configuration file for Getkw though (this is what's slowing down #96, actually)
